packages/snap-preact-components/src/components/Organisms/Sidebar/Sidebar.tsx

```diff
@@ -89,16 +89,16 @@ export const Sidebar = observer((properties: SidebarProps): JSX.Element => {
 
 	return (
 		<CacheProvider>
-			<div {...styling} className={classnames('ss__sidebar')}>
-				{!hideTitle && <h4 className="ss__sidebar__title">{title}</h4>}
+			<div {...styling} className={classnames('ss__sidebar', className)}>
+				{!hideTitle && <h4 className="ss__sidebar__title">{titleText}</h4>}
```

Comment:
This shown when the page is loading, might want to wrap the entire thing in some check?